_translations/en.yml

```diff
@@ -586,9 +586,9 @@ en:
     title: "Bitcoin - Open source P2P money"
     metadescription: "Bitcoin is an innovative payment network and a new kind of money. Find all you need to know and get started with Bitcoin on bitcoin.org."
     listintro: "Bitcoin is an innovative payment network and a new kind of money."
-    list1: "Fast peer-to-peer<br>transactions"
-    list2: "Worldwide<br>payments"
-    list3: "Low<br>processing fees"
+    list1: "Direct person-to-person<br>transactions"
+    list2: "Borderless<br>payments"
+    list3: "No chargebacks<br>(consumer protection still possible)"
```

Comment:
I don't really see how consumer protection is still possible as it requires off-network trust to get a refund.